src/Adapter/Product/Combination/Update/CombinationStockProperties.php

```diff
@@ -62,35 +63,36 @@ class CombinationStockProperties
     private $availableDate;
 
     /**
-     * @param int|null $quantity
+     * @param DeltaQuantity|null $deltaQuantity
      * @param int|null $minimalQuantity
      * @param string|null $location
      * @param int|null $lowStockThreshold
      * @param bool|null $lowStockAlertEnabled
      * @param DateTimeInterface|null $availableDate
      */
     public function __construct(
-        ?int $quantity = null,
+        ?DeltaQuantity $deltaQuantity = null,
```

Comment:
Could you document BC Break ?

Reply:
yep, thanks for heads up. Updated description :bow: